src/vl-wizard-pane.js

```diff
@@ -0,0 +1,205 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/vl-ui-grid/dist/vl-grid.js';
+import '/node_modules/vl-ui-action-group/dist/vl-action-group.js';
+import '/node_modules/vl-ui-button/dist/vl-button.js';
+
+/**
+ * VlWizardPane
+ * @class
+ * @classdesc Gebruik de wizard pane om een stap in de wizard voor te stellen.
+ * 
+ * @extends VlElement
+ * 
+ * @property {boolean} data-vl-next-pane-disabled - Attribuut zorgt ervoor dat de gebruiker niet verder kan naar de volgende stap.
+ * @property {boolean} data-vl-previous-pane-disabled - Attribuut zorgt ervoor dat de gebruiker niet terug kan naar de vorige stap.
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-wizard/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-wizard/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-wizard.html|Demo}
+ * 
+ */
+export class VlWizardPane extends VlElement(HTMLElement) {
+    constructor() {
+        super(`
+            <style>
+                @import '/src/style.css';
+                @import '/node_modules/vl-ui-action-group/dist/style.css';
+                @import '/node_modules/vl-ui-grid/dist/style.css';
+                @import '/node_modules/vl-ui-button/dist/style.css';
+
+                :host {
+                    width: 100%;
+                }
+
+                slot[name="previous-action"], slot[name="next-action"] {
+                    display: inline-block;
+                }
+            </style>
+            <section class="vl-wizard__pane">
+                <slot name="title"></slot>
+                <div is="vl-grid" is-stacked>
+                    <div is="vl-column" size="12">
+                        <slot name="content"></slot>
+                    </div>
+                    <div is="vl-column" size="12">
+                        <div is="vl-action-group">
+                            <slot name="previous-action"></slot>
+                            <slot name="next-action"></slot>
+                        </div>
```

Comment:
voor mij horen de knoppen in de wizard en niet in de panels.

Reply:
Duplicatie opmerking cfr. issue #2.